railib/api.py

```diff
@@ -537,6 +575,11 @@ def query(ctx: Context, database: str, engine: str, command: str,
     tx = Transaction(database, engine, readonly=readonly)
     return tx.run(ctx, _query_action(command, inputs=inputs))
 
+def query_async(ctx: Context, database: str, engine: str, command: str,
```

Comment:
is there no corresponding function for checking status because we are still sync in the first phase?